localization/react-intl/src/app/components/drawer/Projects/ProjectsComponent.json

```diff
@@ -17,23 +17,38 @@
   {
     "id": "projectsComponent.allItems",
     "description": "Label for the 'All items' list displayed on the left sidebar",
-    "defaultMessage": "All items"
+    "defaultMessage": "All"
   },
   {
     "id": "projectsComponent.tiplineInbox",
     "description": "Label for a list displayed on the left sidebar.",
-    "defaultMessage": "Tipline inbox"
+    "defaultMessage": "Inbox"
   },
   {
     "id": "projectsComponent.importedReports",
     "description": "Label for a list displayed on the left sidebar.",
-    "defaultMessage": "Imported fact-checks"
+    "defaultMessage": "Imported"
```

Comment:
The string description is worthy being updated here as the translators need to understand what is imported, especially when translating for gendered languages.

Reply:
I changed this and a few others. I think its better, but let me know if you think a different kind of description would be better.